cmd/helm/template_test.go

```diff
@@ -121,6 +121,12 @@ func TestTemplateCmd(t *testing.T) {
 			wantError: true,
 			golden:    "output/template-with-invalid-yaml-debug.txt",
 		},
+		{
+			name:      "chart with template with unknown field",
+			cmd:       fmt.Sprintf("template '%s' --validate --output-dir '%s'", "testdata/testcharts/chart-unknown-field", "output/dump"),
```

Comment:
For this test to pass, we need a server-side test, I wonder if it'll be easier to have a test which talks to a cluster instead? ( or something similar )

Reply:
It seems that there is no server-side test, and I am not very familiar with the code. I don't know how to add the corresponding test force.
Is it possible to merge the commit of the code repair first, and then add test cases?
```
Any values that would normally be looked up or retrieved in-cluster will be
faked locally. Additionally, none of the server-side testing of chart validity
(e.g. whether an API is supported) is done.
```

Reply:
@bacongobbler Hi, any thoughts on this?